lib/modules/datasource/azure-pipelines-tasks/built-in.ts

```diff
@@ -0,0 +1,153 @@
+export const BuiltInAzurePipelinesTaskMajorVersions: Map<string, string[]> =
```

Comment:
Where does this list come from and how should it be maintained in future?

Reply:
They come from [this directory](https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks). I'm trying to identify an API that will provide the same data.

Reply:
@rarkins I wrote a script to keep this up-to-date #16949 16949